### PR TITLE
Introduction to CNN with TensorFlow

### PR DESCRIPTION
**Pull request by gblasd**
- Neuronal Networks with Keras
- Introduction to Deep Learning and CNN
- Softmax activation
- Embeddings

### CODE REVIEW

**Review by gblasd — COMMENTED** [review]
LGMT!
